lib/screens/release_health/release_health.dart

```diff
@@ -290,13 +283,17 @@ class HealthCard extends StatelessWidget {
   final dangerThreshold = 98;
 
   Color getColor() {
-    return value > warningThreshold
-        ? Color(0xFF23B480)
-        : value > dangerThreshold ? Color(0xFFFFC227) : Color(0xFFEE6855);
+    return value == null
+        ? Color(0xFFB9C1D9)
+        : value > warningThreshold
+          ? Color(0xFF23B480)
+          : value > dangerThreshold ? Color(0xFFFFC227) : Color(0xFFEE6855);
   }
 
   String getIcon() {
-    return value > warningThreshold
+    return value == null
+      ? null
+      : value > warningThreshold
```

Comment:
is this right? if its null, keeps it null, if so, we could use the operator `??`

Reply:
Just wanted to avoid accessing if null, but you are right, there is an obvious way to do this.

Reply:
Hmm, not too sure how to do this. This is basically an early return if value is null.

Using `return value ??` would return the value if not null, which is not what i want. And i only know the following usage of `??=` from the dart doc: 

```
// Assign value to b if b is null; otherwise, b stays the same
b ??= value;
```

Not sure how both would help me here, or i am missing something obvious. 🤔

Reply:
I think the problem is where you placed the statements:

`return value == null ? null : value > warningThreshold`

I understand from that, if `value` is null, set it `null` again.
if `value` is not null, then execute `value > warningThreshold`

so if `value` is null by default, it will never execute the 2nd statement

Reply:
Hmm, in not mutating value here, just returning a null icon name from this method based on the content of value. I will do an early return in the null case to make the intent more clear. Or i am just not seeing it as i have 🍅🍅  on my 👀 (Austrian Proverb).